lya/data.py

```diff
@@ -29,34 +30,38 @@ def load_lya_data(filename=lya_data_path):
 
     data = Data(x, y, y_unc)
 
-    return data
-
-
-def plot_pk(params=None, filename=lya_data_path):
-    """Plot p(k) power spectrum data and theory, with panel for residuals
-    
-    Parameters
-    ----------
-    params : dict or None
-        Cosmological parameter dictionary; if None, then plot only data
-
 
-    Returns
-    -------
-    fig : matplotlib Figure instance
-    """
+    return data
 
-    data = load_lya_data(filename)
+def plot_pk(params=None, filename=lya_data_path, color='k',marker='o',ms=5,ls='',**kwargs):
+"""Plot p(k) power spectrum data and theory, with panel for residuals
```

Comment:
The indentation of this function is missing.  Indentation matters in python (the whole function should be indented; standard is 4 spaces (your text editor should map the tab key to a four-space indent)